crates/holochain_conductor_api/src/docs.rs

```diff
@@ -0,0 +1,63 @@
+//! # Additional Documentation
+//!
+//! ## Enum Serialization Convention
+//!
+//! Serialization of enums exposed on the conductor API in most cases follow the following
+//! convention:
+//!
+//! 1. enums with **only unit-like variants** have their variant names converted to `snake_case`
```

Comment:
Are there still cases that don't adhere to the convention?
```suggestion
//! Serialization of enums exposed on the conductor API follow a convention.
//!
//! ## Enum Serialization Convention
//!
//! 1. Enums with **only unit-like variants** have their variant names converted to `snake_case`
```

Reply:
yes, the enums exposed through the state dump.